src/test/java/study/step3/AmountTest.java

```diff
@@ -0,0 +1,15 @@
+package study.step3;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import study.step3.domain.Amount;
+
+public class AmountTest {
+	@Test
+	@DisplayName("유효성 테스트")
+	public void valid() {
+		Assertions.assertThrows(IllegalArgumentException.class, () -> new Amount(-1));
```

Comment:
아래처럼 메시지와 함께 작성해볼 수 있겠네요.

```suggestion
		assertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {
			Amount amount = new Amount(500);
		}).withMessageMatching("유효하지 않는 금액입니다.");
```